packages/chip/src/Chip/Chip.tsx

```diff
@@ -70,31 +64,25 @@ export const Chip = React.forwardRef<HTMLSpanElement, ChipProps>(
         ref={forwardedRef}
         aria-disabled={disabled}
         className={cx(
-          chipWrapperBaseStyle,
-          chipWrapperThemeStyle(variant, theme),
-          chipWrapperSizeStyle(baseFontSize),
-          {
-            [cx(
-              chipWrapperDisabledStyle[theme],
-              chipWrapperBaseDisabledStyles,
-            )]: disabled,
-          },
+          getChipWrapperStyles(baseFontSize, variant, theme, disabled),
           className,
```

Comment:
could take this 1 step further and pass `className` as an arg so the `@leafygreen-ui/emotion` dependency is completely separated in `*.styles.ts` files